scripts/ipfs/ipfs-swarm-cjdns.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Wait for ipfs to initalize
+attempts=15
+until [[ $(curl http://localhost:5001/api/v0/id -q 2>/dev/null) || ${attempts} -eq 0 ]]; do
+    sleep 1
+    attempts=$((attempts-1))
+done
+
+if [[  ${attempts} -eq 0 ]]; then
+    exit 1
+fi
+
+while read -r cjdns_peer; do
+    cjdns_addr=$(sudo /opt/cjdns/publictoip6 $cjdns_peer)
+
+    # See if they have IPFS enabled
+    res=$(curl http://[${cjdns_addr}]/nodeinfo.json)
+    if [ ! -x "$res" ]; then
+        id=$(echo ${res} | jq -r -M '.services.ipfs.ID')
+        # Value is found
+        if [[ ! ${id} == "null" ]] && [[ ! "${id}" == "" ]]; then
+            # Add them as a bootstrap peer
```

Comment:
I'm being annoying, but *technically* they are being connected to, not added as a bootstrap peer.